src/main/java/com/google/api/codegen/transformer/PageStreamingTransformer.java

```diff
@@ -90,6 +90,14 @@ private PageStreamingDescriptorClassView generateDescriptorClass(
 
     desc.requestTokenSetFunction(
         namer.getFieldSetFunctionName(pageStreaming.getRequestTokenField()));
+    desc.requestPageSizeSetFunction(
```

Comment:
It is more readable if you do:

if (pageStreaming.hasPageSizeField()) {
  desc.requestPageSizeSetFunction(namer.getFieldSetFunctionName(pageStreaming.getPageSizeField())
}